mobile/src/profile/profile.tsx

```diff
@@ -30,7 +29,8 @@ export class Profile extends Component<ProfileProps> {
         const tiles = this.getTiles(employee);
         const contacts = this.getContacts(employee);
 
-        return <View style={styles.container}>
+        return (
+        <ScrollView style={layoutStyles.scrollView}>
```

Comment:
formatting got screwed. Any why parenthesis?

Reply:
Because you can't just move element to the new line without wrapping it

Reply:
@santee ?